op-node/rollup/types.go

```diff
@@ -50,7 +50,29 @@ type Genesis struct {
 	SystemConfig eth.SystemConfig `json:"system_config"`
 }
 
+type ForkName string
+
+const (
+	Regolith ForkName = "regolith"
+	Canyon   ForkName = "canyon"
+	Delta    ForkName = "delta"
+	Ecotone  ForkName = "ecotone"
+	Fjord    ForkName = "fjord"
+	Interop  ForkName = "interop"
+	None     ForkName = "none"
```

Comment:
none can be "bedrock" as the default/initial fork

Reply:
The way I have written it, `nextFork == None` indicates the chain is on the latest known hardfork. Maybe there is a less confusing name? `bedrock` would be the opposite because its the first known hardfork. Looking at `rollup.Config` it seems like all chains must have `bedrock` activated because there is no `BedrockTime` field

Reply:
I ended up changing the new `chainSpec` field to `currentFork` instead of `nextFork`. Should be less confusing. `currentFork == ""` when uninitialized, then it gets set in the first call to `CheckForkActivation`. After that, `CheckForkActivation` is just looking for the appropriate nextFork to see if it gets activated in the current L2 block